src/sonic-yang-models/ietf-inet-types.yang

```diff
@@ -0,0 +1,457 @@
+module ietf-inet-types {
```

Comment:
Since we had dependency with ietf-inet-types(it has been imported in one of the sonic yang) I put it here. Where will the sonic yang fetch the dependencies from if it is not here. Else we can make a common folder and put all the dependent yangs into that. Just like how we have don in mgmt-framework

Reply:
I did not get it, you want to maintain own copy of ietf types. how will you sync it with any ietf yang updates?
We also use IETF definition, but we do not have our own copy of ietf yangs.

Reply:
Removed the ietf yang models